setup.py

```diff
@@ -24,10 +24,12 @@
     'Programming Language :: Python :: 3.5',
     'Programming Language :: Python :: 3.6',
     'Programming Language :: Python :: 3.7',
+    'Programming Language :: Python :: 3.8',
```

Comment:
do we still support <3? if not should we remove these classifiers?

Reply:
yes we still do at this moment, next releases will drop it